.github/workflows/docs.yml

```diff
@@ -0,0 +1,41 @@
+name: Documentation Checks
+
+on: [push]
+
+jobs:
+  spell_check:
+    strategy:
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest]
+    
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Spell check install
+      run: curl -L https://git.io/misspell | bash
+    - name: Spell check docs
+      run: bin/misspell -error docs/*
```

Comment:
Should we run misspell also against codebase?

Reply:
sometimes it gives false positives on code. and we might use names that are not in English or acronyms so most likely no.

Reply:
Otherwise looks really fine

Reply:
plus documentation is more important.

Reply:
> sometimes it gives false positives on code. and we might use names that are not in English or acronyms so most likely no.

yeah that's okay. Our pre-commit hook (which I use) runs misspell against all non git-ignored files, but it can be fixed at some point if needed.